### PR TITLE
implemented file access buffered in core

### PR DESCRIPTION
**Pull request by jeronimo-schreyer**
added write access in file access buffered

### CODE REVIEW

File: core/io/file_access_buffered_fa.h [hunk 1]
```diff
@@ -50,20 +50,34 @@ class FileAccessBufferedFA : public FileAccessBuffered {
 
 		} else {
 
-			cache.offset = p_offset;
-			cache.buffer.resize(p_size);
+			if (read_cache.buffer.size() == 0) {
+                read_cache.buffer.resize(cache_size);
```
Comment (akien-mga) [inline core/io/file_access_buffered_fa.h]: Indentation issues.

**Comment by aaronfranke** [comment]
The repository this PR came from has been deleted, so this PR is no longer valid.

**Comment by jeronimo-schreyer** [comment]
sorry, I thought this was already merged. The repository from where this came from was mine but forked myself a new one. Write buffer should be enabled to optimize write access. This is something that is not mandatory on PC but there are some devices that limit its write access

**Comment by aaronfranke** [comment]
@jeronimo-schreyer Feel free to open another PR with the changes, this one is dead though.

**Pull request closed without merging**